spec/spec_helper.rb

```diff
@@ -27,4 +29,5 @@
   c.default_facts = default_facts
 end
 
+add_custom_fact :pe_version, '2016.4.2'
```

Comment:
can you move this to a `default_module_facts.yml`? https://github.com/voxpupuli/puppet-r10k/blob/master/spec/spec_helper.rb#L26